__mocks__/mapDefinitions/NestedLoopMapDefinition.ts

```diff
@@ -1,20 +0,0 @@
-export const nestedLoopMapDefinition = `$version: 1
```

Comment:
Comprehensive already has a nested loop example in it? I don't see any update here to add one

Reply:
Correct, it covers all of our base scenarios already (including ManyToMany, ManyToOne, etc)